### PR TITLE
[Fixes #72] Add timeline and impact message boxes

### LINKED ISSUE #72: Timeline and Impact Activity Message Boxes
For Activity pages, we should have ave the following highlighted at the top:
1. Timeline - how long the activity will last
2. Impact - why this is an important activity right now and the potential impact it can have.

I imagine these to be CSS boxes with different styling and a border applied, but am open to suggestions on how it is best implemented.

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Comment by comzeradd** [comment]
Here is how it looks like

shot

**Comment by brianking** [comment]
Looks good to me.

File: _pages/es/test-pilot.md [hunk 1]
```diff
@@ -11,6 +11,16 @@ redirect_from: "/es/experiments/test-pilot/"
 * TOC
 {:toc}
 
+{: .label .timebox}
+<span class="glyphicon glyphicon-time" aria-hidden="true"></span> 2 Hours
```
Comment (MichaelKohler) [inline _pages/es/test-pilot.md]: I would write "hours" with a lowercase h here.

Comment (MichaelKohler) [inline _pages/es/test-pilot.md]: (even though this is es)

File: _pages/test-pilot.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ redirect_from: "/experiments/test-pilot/"
 * TOC
 {:toc}
 
+{: .label .timebox}
+<span class="glyphicon glyphicon-time" aria-hidden="true"></span> 2 Hours
```
Comment (MichaelKohler) [inline _pages/test-pilot.md]: Would you mind exchanging this with "2 hours" (lowercase h) to make it consistent with the other pages?

**Comment by comzeradd** [comment]
Thanks @MichaelKohler. Good catch. I used lowercase for hours everywhere.

**Pull request merged**